docker/config/app.env

```diff
@@ -0,0 +1,8 @@
+# Environment variables for local dev environment
+
+REDIS_HOST=ichnaea_redis_1
+
+# Database bits
+DB_HOST=ichnaea_mysql_1
```

Comment:
These are probably wrong settings. They should be the service names which are exposed as host names inside the containers.